src/containers/BooksList.js

```diff
@@ -0,0 +1,53 @@
+import React from 'react';
+import { connect } from 'react-redux';
+
+import { removeBook } from '../actions/index';
+import Book from '../components/Book';
+
+class BooksList extends React.Component {
+  constructor(props){
+    super(props)
+    this.handleRemoveBook = this.handleRemoveBook.bind(this);
+  }
+
+  handleRemoveBook(book) {
+    this.props.removeBook(book)
+  }
```

Comment:
[Optional] This method is unnecessary. We could pass `this.props.removeBook` to the Book component directly.